experiment_buddy/experiment_buddy.py

```diff
@@ -249,40 +251,37 @@ def _open_ssh_session(hostname: str) -> fabric.Connection:
     return ssh_session
 
 
-def _ensure_scripts(hostname: str, extra_slurm_header: str, working_dir: str) -> Tuple[str, fabric.Connection]:
-    ssh_session = _open_ssh_session(hostname)
+def ensure_scripts(ssh_session: fabric.Connection, extra_slurm_header: str, working_dir: str) -> str:
     retr = ssh_session.run("mktemp -d -t experiment_buddy-XXXXXXXXXX")
-    remote_tmp_folder = retr.stdout.strip() + "/"
+    remote_tmp_folder = f'{retr.stdout.strip()}/'
+    ssh_session.put(f'{SCRIPTS_PATH}/common/common.sh', remote_tmp_folder)
```

Comment:
maybe common/* ?

Reply:
maybe you need a for loop or some flag for globs

Reply:
I have only a file right now and I hope it will remains one single file 

The idea was to put it directly in $SCRIPTS_PATH  but, I don't know exactly why, I had some issues using it directly from there

For now I'd leave it explicit, given that the directory structure is different from the repo